app/views/layouts/_navbar.html.erb

```diff
@@ -1,16 +1,15 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+<nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark h5">
```

Comment:
`fixed-top` is necessary?  Flash messages appear under the navbar so we don't see them.

Reply:
No, I changed that because you mentioned it I think. Flash messages are not affected by margins?